connectors/sources/network_drive.py

```diff
@@ -112,16 +107,19 @@ async def _remote_validation(self, advanced_rules):
                 is_valid=False,
                 validation_message=e.message,
             )
+        start_with_slash = set()
 
-        await asyncio.to_thread(self.source.smb_connection.create_connection)
-
-        _, invalid_rules = await self.source.find_matching_paths(advanced_rules)
+        for rule in advanced_rules:
+            pattern = rule["pattern"]
+            if pattern.startswith("/"):
+                start_with_slash.add(pattern)
```

Comment:
Is this sufficient glob validation? What about special characters like "$" and stuff, do these need to be escaped?

Reply:
yes, we have verified with all these special characters: `C:\Users\Administrator\Documents\perf-data-1lac\LatestData\home\runner\!work#$%^&&&^$^&^&(&^&+_)` and it seems to be working fine with this. The only validation needs to be add is that path should not start with a slash.